src/pynxtools/nomad/parser.py

```diff
@@ -491,18 +491,25 @@ def parse(
                 app_sec = getattr(self.nx_root, app)
                 try:
                     app_entry = getattr(app_sec, "ENTRY")
+                    if len(app_entry) < 1:
+                        raise AttributeError()
                 except AttributeError:
                     app_entry = getattr(app_sec, "entry")
+                    if len(app_entry) < 1:
+                        raise AttributeError()
```

Comment:
A quick question, app_entry is not being used anywhere. Or is it to use in the future?

Reply:
it is only to check if exists, so if no real entry is found then we do not add a corresponding AppDef to the entry_type

Reply:
Right! I missed line:499 Attribute error.